packages/offix-datastore/tests/mock.graphql

```diff
@@ -0,0 +1,21 @@
+""" @model """
+type Note {
+    id: ID!
+    title: String!
+    description: String!
+    """
+    @oneToMany field: 'noteComment', key: 'noteId'
```

Comment:
Should this be `key: 'commentId'`, rather than `noteId` or whatever the equivalent is in the db?

Reply:
I just copied this file from graphback to use for testing. Do we need to focus on the `@oneToMany` annotation? I'm not using it. At least not now.

Reply:
No don't stress about it. I'm just going through the code and started with the models to get an idea of what is going on.

Looks like you have done an incredible amount of work.

Were you able to solve the db concurrency?

Reply:
Yes. The problem came from db connections I opened directly in the tests, these connections didn't have an `onversionchange` handler so they had to be closed manually or they block any upgrade requests